src/mlx/warnings.py

```diff
@@ -214,7 +216,7 @@ def warnings_wrapper(args):
     # Read config file
     if args.configfile is not None:
         checkersflag = args.sphinx or args.doxygen or args.junit or args.coverity or args.xmlrunner
-        if checkersflag or (args.maxwarnings != 0) or (args.minwarnings != 0):
+        if checkersflag or (args.maxwarnings | args.minwarnings | args.exact_warnings):
```

Comment:
I liked `!=0` more as that is more explicit. We could combine this into `warningsflag` variable...

Reply:
650c06b7e47df067ee5f1a1cb666cecf56b79b95

Reply:
CodeClimate doesn't like your suggestion. It says "Consider simplifying this complex logical expression."

Reply:
Ok, I fixed that.